components/scream/src/interface/atm_comp_mct.F90

```diff
@@ -204,7 +209,7 @@ end subroutine scream_run
          infodata=infodata)
 
     call dead_run_mct('atm', EClock, x2d, d2x, &
-       gsmap, ggrid, gbuf, mpicom, compid, my_task, master_task, logunit)
+       gsmap, ggrid, gbuf, mpicom_atm, compid, my_task, master_task, logunit)
```

Comment:
we're eventually going to need to make this work without mct assuming we have a 'dead atm', right?

Reply:
Yes, for now we are working within the context of a "dead atm component" but the syntax should be the same when we switch to a standard interface.  In the context of this PR, we will still need to gather the mpicom for the atm component from the component coupler in the final product.